Source/JavaScriptCore/runtime/TemporalDuration.cpp

```diff
@@ -94,10 +94,8 @@ ISO8601::Duration TemporalDuration::fromDurationLike(JSGlobalObject* globalObjec
         JSValue value = durationLike->get(globalObject, temporalUnitPluralPropertyName(vm, unit));
         RETURN_IF_EXCEPTION(scope, { });
 
-        if (value.isUndefined()) {
-            result[unit] = 0;
+        if (value.isUndefined())
             continue;
```

Comment:
This is just a cleanup, is my understanding correct? (Since Duration is initialized with zeros).

Reply:
Yeah, precisely. Since this function is almost identical to the other one, I realized that we could remove this line.